### PR TITLE
Chapter 27

### PR DESCRIPTION
**Pull request by StanHash**
~~*Finally* I got this to be rebased properly~~ Chapter 27 Stuff!

Should be fully functional. The only thing I didn't test is the last minute change to the `move to next chapter` event instruction (I had it send the player to chapter 1 during testing).

### CODE REVIEW

File: Engine Hacks/_MasterHackInstaller.event [hunk 1]
```diff
@@ -152,6 +152,13 @@ POP
   FreezeAllASMC:
   #incbin "ASM/freezeall.dmp"
 
+  // Chapter 27 ASMC-able functions
+  ALIGN 4
+  gChapter27FogLevel:
+    WORD Ch27FogLevel
+  
+  #include "CCBlitzChapter27Asmcs.lyn.event"
```
Comment (Crazycolorz5) [inline Engine Hacks/_MasterHackInstaller.event]: Including the output of lyn rather than calling lyn directly?

File: Events/Ch27.event [hunk 2]
```diff
@@ -0,0 +1,1372 @@
+#ifndef CCBLITZ_STAN_CH27
+#define CCBLITZ_STAN_CH27
+
+/*
+
+I hope y'all like Fog of war Desert Maps with one-tile hidden treasure =^)
```
Comment (Crazycolorz5) [inline Events/Ch27.event]: I don't

File: Engine Hacks/_MasterHackInstaller.event (hunk 1 above)
Comment (StanHash) [reply Engine Hacks/_MasterHackInstaller.event]: Given that the lyn *input* (the compiled object file) is already generated I feel like there'd be no gain in doing that (If we could include from source that'd be another story, but sadly here we do not do that).

File: Events/Ch27.event (hunk 2 above)
Comment (StanHash) [reply Events/Ch27.event]: A shame

**Pull request merged**